ngraph/python/CMakeLists.txt

```diff
@@ -16,30 +16,71 @@
 
 cmake_minimum_required (VERSION 3.13)
 
-project (pyngraph)
-
-include(ExternalProject)
-
-ExternalProject_Add(
-        pybind11
-        GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
-        GIT_TAG "v2.5.0"
-        SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/pybind11"
-        CONFIGURE_COMMAND ""
-        BUILD_COMMAND ""
-        INSTALL_COMMAND ""
-)
+project (_pyngraph)
 
-set(BUILD_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/build_wheel.py.in")
-set(BUILD_PY    "${CMAKE_CURRENT_BINARY_DIR}/build_wheel.py")
-configure_file(${BUILD_PY_IN} ${BUILD_PY} @ONLY)
+include(FetchContent)
 
-add_custom_command(
-        DEPENDS pybind11
-        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/dist/
-        POST_BUILD
-        WORKING_DIR ${CMAKE_CURRENT_BINARY_DIR}
-        COMMAND python ${BUILD_PY}
+FetchContent_Declare(
+  pybind11
+  GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
+  GIT_TAG        "v2.5.0"
 )
 
-add_custom_target(python_wheel DEPENDS ngraph ${CMAKE_CURRENT_BINARY_DIR}/dist/)
+FetchContent_GetProperties(pybind11)
+if(NOT pybind11_POPULATED)
+  FetchContent_Populate(pybind11)
+  add_subdirectory(${pybind11_SOURCE_DIR} ${pybind11_BINARY_DIR})
+endif()
+
+find_package(ngraph REQUIRED)
```

Comment:
What if cmake find the system nGraph?

Reply:
system nGraph will be used in this case. Anyway refactored to take into consideration CMake working directory.